tests/e2e/footer.spec.ts

```diff
@@ -68,10 +68,10 @@ const footerSocialMediaLinks = [
     linkName: 'Youtube',
     linkAddress: 'https://www.youtube.com/channel/UCEULduld5NrqOL49k1KVjoA/',
   },
-  {
-    linkName: 'Instagram',
-    linkAddress: 'https://www.instagram.com/kodadot.xyz/',
-  },
+  // {
+  //   linkName: 'Instagram',
+  //   linkAddress: 'https://www.instagram.com/kodadot.xyz/',
+  // },
```

Comment:
let's fix it in another PR or maybe @prury has an idea?

Reply:
> let's fix it in another PR or maybe @prury has an idea?

can we leave it like that and I'll apply a proper fix in another PR?